Source/Utils/WarpXAlgorithmSelection.H

```diff
@@ -114,6 +114,16 @@ struct ParticleBoundaryType {
     };
 };
 
+/** MPI reductions
+ */
+struct ReductionType {
+    enum {
+        Maximum = 0,
+        Minimum = 1,
+        Sum = 2
+    };
+};
+
```

Comment:
I don't know if this belongs in `WarpXAlgorithmSelection` (it feels overly specific) but I have to admit that these structs are really convenient.

Reply:
Yes, ok for now unless we have a better idea :)